chronologger.go

```diff
@@ -4,58 +4,41 @@ import (
 	"context"
 	"time"
 
-	"github.com/jaswdr/faker"
 	"github.com/labstack/echo/v4"
-	"go.mongodb.org/mongo-driver/bson"
-	"go.mongodb.org/mongo-driver/mongo"
-	"go.mongodb.org/mongo-driver/mongo/options"
+
+	"github.com/hay-i/chronologger/db"
+	"github.com/hay-i/chronologger/models"
 )
 
-func getAddress(c echo.Context) error {
-	fake := faker.New()
-	title := fake.Address().Address()
-	component := address(title)
+func getTemplates(c echo.Context, templates []models.Template) error {
+	component := buildTemplates(templates)
 
 	return component.Render(c.Request().Context(), c.Response().Writer)
 }
 
 func main() {
 	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
-	defer cancel()
-
-	clientOpts := options.Client().ApplyURI("mongodb://root:example@localhost:27017")
-	client, err := mongo.Connect(ctx, clientOpts)
-
+	client, dbErr := db.Initialize(ctx)
+	database := client.Database("chronologger")
 	defer func() {
-		if err = client.Disconnect(ctx); err != nil {
-			panic(err)
+		if dbErr = client.Disconnect(ctx); dbErr != nil {
+			panic(dbErr)
 		}
 	}()
+	defer cancel()
 
-	if err != nil {
-		panic(err)
-	}
-
-	collection := client.Database("chronologger").Collection("goals")
-	fake := faker.New()
-	title := fake.Address().Address()
-	goal := Goal{Title: title, CreatedAt: time.Now()}
-	collection.InsertOne(ctx, goal)
-
-	var result Goal
-	findOptions := options.FindOne().SetSort(bson.D{{Key: "created_at", Value: -1}})
-	collection.FindOne(ctx, bson.D{}, findOptions).Decode(&result)
-	// result isn't used anymore, but this can be used as an example for the moment
+	db.Seed(ctx, database)
```

Comment:
Part of me is wondering if we put this in a script in hind-sight. It'll only have to be ran once per instance (local or prod) so it could just sit as a script and only ran when needed... not sure

Reply:
I'll build the project and everything from scratch tomorrow and let you know how it feels